extensions/amp-story/1.0/amp-story-system-layer.css

```diff
@@ -134,7 +131,7 @@
   display: flex !important;
   height: 2px !important;
   left: 0 !important;
-  margin: 4px 0 0 !important;
+  margin: 6px 10px 0 10px !important;
```

Comment:
When does the button get the active attribute?
I'm searching the codebase and not seeing it. I may be missing it, or maybe this is dead code?

Reply:
I think this comment refers to the `:active`, which is the state when you click them (it's an HTML pseudo-selector).

Check the dark background while clicking:
![image](https://user-images.githubusercontent.com/22420856/144923405-76b588e6-f032-4ea2-9013-3795067f33d7.png)